<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_screen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>20</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 25 20pt &quot;Ubuntu&quot;;
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border-style: solid;
	border-width: 1px;
	border-radius: 2px;
	border-color: #758ca0;
	margin: 1px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sciMode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>Sci.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>clr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9nine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7seven">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8eight">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6six">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4four">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5five">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3three">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subtract">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1one">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2two">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0zero">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #388ed8;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #276fad;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #2574ba;
	color: #afd9ff;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decimal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_help">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e8ebef;
    border-style: solid;
    border-width: 1px;
    border-radius: 2px;
    border-color: #758ca0;
	margin: 1px;
}
QPushButton:pressed {
    background-color: #388ed8;
}
QPushButton:hover{ 
	background-color: #b5bfcc;
	color: #374c5e;
}
</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
